src/components/graphs/PieChart.vue

```diff
@@ -1,12 +1,21 @@
 <template>
   <div class="pie-chart-cont">
     <svg id="pie-chart"><!-- D3 inserts here --></svg>
+    <img
```

Comment:
We need to make sure that the tooltip is accessible to folks using just a keyboard, who navigate websites  by tabbing through it (with the Tab key). Add a `tabindex="0"` and it'll be tabbable!

![Screenshot from 2025-02-03 19-24-52](https://github.com/user-attachments/assets/cea60585-fd0f-48c4-93f1-e3805f9d895b)